docs/README.md

```diff
@@ -1,3 +1,8 @@
+---
+title: watsonx Generative AI Lab
+description: Overview and About watsonx Generative AI Lab
```

Comment:
Overview and About watsonx Generative AI Lab --? How to use Generative AI using watsonx Lab